v2v/tests/src/nbdkit/nbdkit.py

```
@@ -695,6 +695,25 @@ def check_curl_time_option():
+        if len(hash_code) != 64:
+            test.fail('fail to test blkhash option')
+
+        hash_option = params_get(params, "hash_option")
+        test_state = params_get(params, "test_state")
+        if test_state == "positive":
+            cmd = process.run("nbdcopy --blkhash=%s -- %s null:" % (hash_option, qcow2_image), shell=True, ignore_status=True)
+            hash_code = cmd.stdout_text.strip()
+            if len(hash_code) != 32:
+                test.fail('fail to test blkhash option positive case')
+        elif test_state == "negative":
+            cmd = process.run("nbdcopy --blkhash=%s -- %s null:" % (hash_option, qcow2_image), shell=True, ignore_status=True)
```

Ruff (0.14.4)

Function call with `shell=True` parameter identified, security issue

(S604)